tests/models/tile_model_tests.py

```diff
@@ -312,6 +312,11 @@ def test_save_provisional_from_athoritative(self):
 
         provisionaledits = provisional_tile.provisionaledits
         self.assertEqual(tiles.count(), 2)
+        self.assertIn(
```

Comment:
Unrelated, but not really worth a whole new PR for it: follow-up to #10860 where I wanted to add some more debug output, but was stymied by the KeyError being raised much earlier.